client/client.go

```diff
@@ -0,0 +1,116 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	xlineapi "github.com/xline-kv/go-xline/api/xline"
+	"github.com/xline-kv/go-xline/xlog"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// Xline client
+type client struct {
+	// Kv client
+	Kv kvClient
+}
+
+func Connect(allMembers []string, options ...ClientOptions) (*client, error) {
+	logger := xlog.GetLogger()
+
+	token := ""
+	clientTimeout := newDefaultClientTimeout()
+
+	// init timeout
+	if len(options) !=0 {
+		clientTimeout = newClientTimeout(options[0].CurpTimeout)
+	}
+
+	// get token
+	if len(options) != 0 && options[0].User.Name != "" && options[0].User.Password != "" {
```

Comment:
Why don't you merge this `if` block with the previous one?

Reply:
Thank you for the feedback. I've updated it as you recommended.